x-pack/plugins/cross_cluster_replication/server/routes/api/auto_follow_pattern.js

```diff
@@ -33,8 +31,6 @@ export const registerAutoFollowPatternRoutes = (server) => {
     handler: async (request) => {
       const callWithRequest = callWithRequestFactory(server, request);
 
-      // throw wrapEsError(esErrors[403]); // Temp for development to test ES error in UI. MUST be commented in CR
```

Comment:
Good that you removed this. I have asked to get a license for "Charles" to be able to test multiple API responses and delay in the UI and avoid this horrible hack. 😊